sql/hive-thriftserver/pom.xml

```diff
@@ -41,11 +41,8 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.apache.spark</groupId>
-      <artifactId>spark-core_${scala.binary.version}</artifactId>
-      <version>${project.version}</version>
-      <type>test-jar</type>
-      <scope>test</scope>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-servlet</artifactId>
```

Comment:
I actually think that's a different, unrelated error that has been noted several times. There's a JIRA I can't find just now but see http://blogs.candoerz.com/question/128074/spark-sql-15-build-failure.aspx for example. That is I don't think there should be a POM change to address this as I understand it to be some env issue.